Sources/APNS/APNS.swift

```diff
@@ -0,0 +1,79 @@
+import Vapor
+
+public struct APNS {
+    struct ConfigurationKey: StorageKey {
+        typealias Value = APNSwiftConfiguration
+    }
+
+    public var configuration: APNSwiftConfiguration? {
```

Comment:
With this change, `Request`'s `APNS` property will now have these properties exposed on it. If that's an issue, I can refactor it to only be available to `Application`!